crates/subspace-transaction-pool/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "subspace-transaction-pool"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Hmm, we don't have a clear written rule about the package info, but you try adding a few more by referring to another crate.